### PR TITLE
Added Status DataType

### PR DESCRIPTION
**Pull request by edx-acooper**
Hey, can you verify changes.

### CODE REVIEW

**Comment by aharper2568** [comment]
good

**Comment by JLH-Owner** [comment]
awesome job!

**Comment by zoniaramirez** [comment]
looks good!

File: models/User.js [hunk 1]
```diff
@@ -35,6 +35,10 @@ User.init(
         len: [8],
       },
     },
+    status: {
+      type: DataTypes.ENUM,
```
Comment (edx-acooper) [inline models/User.js:39]: What the heck is this?

Comment (edx-acooper) [reply models/User.js:39]: @edx-acooper ⬆️

**Review by aharper2568 — COMMENTED** [review]
mystery code @zoniaramirez

**Pull request merged**